frontend/src/layout/navigation/SideBar/SideBar.tsx

```diff
@@ -182,7 +193,12 @@ function Pages(): JSX.Element {
                     identifier: Scene.Insight,
                 }}
             />
-            <PageButton icon={<IconRecording />} identifier={Scene.SessionRecordings} to={urls.sessionRecordings()} />
+            <PageButton
```

Comment:
Doing this as part of this PR feels wrong to me, same with persons & groups. I don't see these tags adding anything but noise and we haven't even discussed this earlier.